oep-template.rst

```diff
@@ -0,0 +1,118 @@
+=====================
+OEP-XXX: OEP Template
+=====================
+
```

Comment:
does OEP = object engineering process? Whatever it stands for, you should write it out in full early on to orient people and get better SEO (search engine optimization :) )